### PR TITLE
fetch_trams usa novo rcongresso

### PR DESCRIPTION
**Pull request by andresmrm**

### CODE REVIEW

**Comment by jairNeto** [comment]
Deu conflitos

**Comment by andresmrm** [comment]
Arrumei

File: R/tramitacoes.R [hunk 1]
```diff
@@ -60,53 +20,49 @@ fetch_tramitacao_senado <- function(proposicao_id, normalized=FALSE) {
...
 
+#' @title Baixa os dados da tramitação da Câmara
+#' @description Retorna dataframe com os dados da tramitação de uma proposição da Camara
+#' @param bill_id ID de uma proposição na Camara
+#' @return Dataframe com os dados da tramitação de uma proposição da Camara
+#' @examples
+#' fetch_tramitacao_camara(2121442, TRUE)
+fetch_tramitacao_camara <- function(bill_id) {
+  rcongresso::fetch_tramitacao_camara(bill_id) %>%
+    rcongresso::rename_df_columns() %>%
```
Comment (jairNeto) [inline R/tramitacoes.R]: Error: 'rename_df_columns' is not an exported object from 'namespace:rcongresso'

Comment (andresmrm) [reply R/tramitacoes.R]: Fiz um PR que deve arrumar isso:
https://github.com/andresmrm/rcongresso/pull/27

**Pull request merged**